python/tank/descriptor/io_descriptor/base.py

```diff
@@ -320,14 +323,16 @@ def _get_locally_cached_versions(self):
                 for version_folder in os.listdir(parent_folder):
                     version_full_path = os.path.join(parent_folder, version_folder)
                     # check that it's a folder and not a system folder
-                    if os.path.isdir(version_full_path) and \
+                    if not self._is_settings_folder(version_full_path) and \
```

Comment:
Doesn't this mean that previous versions of Toolkit will think this is an actual bundle folder? Does this mean we will start reporting extra versions with `.settings` will previous versions of Toolkit? How will this impact the search for the latest version of a bundle in previous versions of Toolkit?